DigitalLearningSolutions.Web/Models/Enums/DelegateGroupTab.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace DigitalLearningSolutions.Web.Models.Enums
+{
+    using System.Collections.Generic;
+
+    public class DelegateGroupTab : BaseTabEnumeration
+    {
+        public static DelegateGroupTab Delegates = new DelegateGroupTab(
+            1,
+            nameof(Delegates),
+            "GroupDelegates",
+            "Index",
+            "Delegates"
+        );
+
+        public static DelegateGroupTab Courses = new DelegateGroupTab(
+            2,
+            nameof(Courses),
+            "GroupCourses",
+            "Index",
+            "Courses"
+        );
+
+        public DelegateGroupTab(int id, string name, string controller, string action, string linkText) : base(

```

Comment:
You could make this constructor private _(unless its going to used outside the class eventually)_